package.json

```diff
@@ -29,6 +29,7 @@
     "plop": "^1.8.1",
     "prettier": "^1.14.3",
     "rimraf": "^2.6.1",
+    "semver": "^5.6.0",
```

Comment:
Should this be a dependency of _all_ packages, or rather a dependency of packages/gatsby, e.g. gatsby core?